Description: Add libvirt-bin alias to libvirtd sysvinit
 To have the same backwards compat alias name for the sysvinit script
 as we have for the systemd service.
Author: Stefan Bader <stefan.bader@canonical.com>
Forwarded: no

Index: libvirt-1.3.4/daemon/libvirtd.init.in
===================================================================
--- libvirt-1.3.4.orig/daemon/libvirtd.init.in	2016-01-12 01:12:06.000000000 +0100
+++ libvirt-1.3.4/daemon/libvirtd.init.in	2016-06-02 15:44:11.151417213 +0200
@@ -4,7 +4,7 @@
 # http://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/initscrcomconv.html
 #
 ### BEGIN INIT INFO
-# Provides: libvirtd
+# Provides: libvirtd libvirt-bin
 # Default-Start: 3 4 5
 # Default-Stop: 0 1 2 6
 # Required-Start: $network messagebus virtlogd
